tests/unittests/TFLiteImporterTest.cpp

```diff
@@ -195,6 +220,8 @@ TFLITE_UNIT_TEST(Sin, "sin.tflite")
 
 TFLITE_UNIT_TEST(Tile, "tile.tflite")
 
+TFLITE_UNIT_TEST_MAX_ERR(ResizeBilinear, "resize_bilinear.tflite", 3e-6)
```

Comment:
Any idea why the error is higher than for the other operators? Thanks!

Reply:
Highest error is 1.9e-6 which I find acceptable. I am simply comparing what tensorflow produced and what Glow produced and given the error, didn't want to explore this further.